application/src/main/java/org/cardanofoundation/ledgersync/service/impl/genesis/GenesisDataServiceImpl.java

```diff
@@ -402,7 +407,14 @@ public void fetchConwayGenesis(GenesisData genesisData) {
                     .drepActivity(convertObjecToBigInteger(genesisConwayJsonMap.get(D_REP_ACTIVITY)))
                     .build();
 
-            if (committee.get(THRESHOLD) != null) {
+            if (committee.get(THRESHOLD) instanceof Map) {
+                final var ccThreshold = (Map<String, Object>) committee.get(THRESHOLD);
+                BigDecimal numerator = convertObjectToBigDecimal(ccThreshold.get(CC_THRESHOLD_NUMERATOR));
```

Comment:
@Sotatek-HuyLe3a Do we need to use BigDecimal? Or, BigInteger is ok ?

Any idea what's the data type for the same in Haskell code?